SU2_CFD/src/solver_direct_mean.cpp

```diff
@@ -7320,71 +7310,51 @@ void CEulerSolver::SetActDisk_BCThrust(CGeometry *geometry, CSolver **solver_con
 void CEulerSolver::SetFarfield_AoA(CGeometry *geometry, CSolver **solver_container,
```

Comment:
This function got cleaned up, and now it does only SetFarfieldAoA (As opposed to perform all the logic for the Fixed CL mode). Moved all that to the FixedCL_Convergence function. Which now that I say out loud, should be name FixedCL_Driver or something of that sort